browser/src/control/jsdialog/Widget.Checkbox.ts

```diff
@@ -18,7 +18,7 @@ function _createCheckboxContainer(
 ) {
 	const container = window.L.DomUtil.create(
 		'div',
-		builder.options.cssClass + ' ui-checkbox checkbutton',
```

Comment:
are we sure it is not used anymore?
good to have it removed to keep all unified but we have to be careful